nyxx.interactions/pubspec.yaml

```
@@ -0,0 +1,26 @@
+name: nyxx_interactions
+version: 1.1.0-dev.2
+authors:
+  - HarryET
+description: Interactions for Nyxx library
+homepage: https://github.com/l7ssha/nyxx
+repository: https://github.com/l7ssha/nyxx
+documentation: https://github.com/l7ssha/nyxx/wiki
+issue_tracker: https://github.com/l7ssha/nyxx/issue
+
+environment:
+  sdk: '>=2.12.0-51.0.dev <3.0.0'
+
+dependencies:
+  logging: "^1.0.0-nullsafety.0"
+  nyxx: "^1.1.0-dev.2"
```

```suggestion
  nyxx: "^1.1.0-dev.1"
```
I will change it back to to `.2` after release

---

Okay!